### PR TITLE
chore: remove steam layering option

### PR DESCRIPTION
**Pull request by RoyalOughtness**
rationale: reports of layering steam causing issues. we should discourage this option and not officially recommend it. If users want to layer steam, they should do so in their own downstream image. Otherwise, they should distrobox or flatpak it.

### CODE REVIEW

File: files/justfiles/utilities.just [hunk 1]
```diff
@@ -184,16 +184,14 @@ install-steam:
...
-        read -p "Selection [1-3]: " method_selection
-        if [[ "$method_selection" == [123]* ]]; then
+        read -p "Selection [1-2]: " method_selection
+        if [[ "$method_selection" == [12]* ]]; then
             valid_input="1"
         else
             echo "That is not a valid selection."
         fi
     done
-    valid_input="0"
```
Comment (RoyalOughtness) [inline files/justfiles/utilities.just:196]: @spaceoden fyi i removed this as well since it doesn't have any effect

Comment (spaceoden) [reply files/justfiles/utilities.just:196]: oops. good catch.

**Review by spaceoden — APPROVED** [review]
lgtm. also we can consider reverting #663 because it was done specifically to assist with steam deps

**Comment by RoyalOughtness** [comment]
> lgtm. also we can consider reverting #663 because it was done specifically to assist with steam deps

https://github.com/secureblue/secureblue/pull/716

**Review by EsseLowNitro — APPROVED** [review]

**Pull request merged**